pkg/authorization/api/types.go

```diff
@@ -67,6 +69,7 @@ var (
 		KubeExposedGroupName:   {"pods", "replicationcontrollers", "services", "endpoints", "pods/log"},
 		KubeAllGroupName:       {KubeInternalsGroupName, KubeExposedGroupName, QuotaGroupName},
 		KubeStatusGroupName:    {"pods/status", "resourcequotas/status", "namespaces/status"},
+		KubePodAdminGroupName:  {"pod/exec", "pod/proxy", "pod/portforward"},
```

Comment:
Notwithstanding the problem of privileged containers, can we say that in general, if you have read access to the pod, you can do proxy and port forward, and if you have write/update access you can have exec?